packages/admin/src/utils/callServiceMethod.ts

```diff
@@ -78,7 +78,7 @@ export default async function callServiceMethod<
     if (err.code) {
       return generateErrorPageRedirectV2(err.code, errorPageParams as string);
     }
-
+    console.log(err);
```

Comment:
is this intended?

Reply:
no although it does actually make debugging this quite a lot easier lol, will remove